lib/setup.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+  'use strict';
+
+  var pytools = require('@optbot/pytools');
+  var shell = require('shelljs');
+  var path = require('path');
+  var nconf = require('nconf');
+  var user = process.env.npm_package_config_user;
+  var configFile = path.join(process.env.npm_config_quichean_nconf_path,
+    'pytools', 'config.json');
+  var venvPath;
+  var venvs;
+  var logPath;
+
+  // create 'eqpuller' user
+  shell.exec(path.join(__dirname, 'createuser.sh') + ' ' + user +
+    ' "--shell /bin/bash"');
+  // set up python virtual environment
+  pytools.init();
+  nconf.file({file: configFile});
```

Comment:
It's not a huge deal, but specifying only "file" is redundant in this case (without more options). There's an overload that just accepts the path as a string.